src/components/TableOfContents/TableOfContentsLink.tsx

```diff
@@ -29,16 +26,20 @@ const Link = ({
   const $dotBg = cssVar("dot-bg")
 
   const hoverOrActiveStyle: SystemStyleObject = {
-    color: "primary.base",
+    color: $dotBg.reference,
     _after: {
       content: `""`,
-      background: $dotBg.reference,
+      backgroundColor: "background.base",
       border: "1px",
-      borderColor: "primary.base",
+      borderColor: $dotBg.reference,
       borderRadius: "50%",
       boxSize: 2,
       position: "absolute",
-      insetInlineStart: "-1.29rem",
+      // 16px is the initial list padding
+      // 8px is the padding for each nested list
+      // 4px is half of the width of the dot
+      // 1px for the border
+      insetInlineStart: `calc(-16px - 8px * ${depth} - 4px - 1px)`,
```

Comment:
@TylerAPfledderer Do you happen to know why this is creeping up again? We import `SystemStyleObject` from `@chakra-ui/react`, and use `insetInlineStart` here, but it ends up compiling to `left`:

<img width="554" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/eecdfa42-2357-4ace-8975-2e2ec76656b3">

Causing this:

<img width="250" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/63340c13-0196-4e75-9baa-d2313b766ae4">

Reply:
@pettinarip @wackerow Chakra compiles `insetInlineStart` to either right or left depending on the global set `dir`.

However, here I'm not sure why it won't compile correctly. Digging into the source code, my guess it that Chakra does not like compiling a logical prop like this when it's in a pseudo. (I can't quite confirm that because the compiling flow is quite dense for me). I do see it working as expected in Storybook. Would need some time to see if there is something coming from an ancestor component affecting this.

A workaround here would be to force the original CSS prop with the kebab case.

```tsx
Before
insetInlineStart: `calc(-16px - 8px * ${depth} - 4px - 1px)`

After
"inset-inline-start": `calc(-16px - 8px * ${depth} - 4px - 1px)`
```

Now it should work in both Storybook and the dev environment.

> I'm not expecting this kind of logic for Chakra v3. Seems like overkill here, and would be able to go back to camelCase.

Reply:
Cool, that worked fine. Pushing this change. Thanks

Reply:
Okay cool, thanks @TylerAPfledderer! If this is indeed related to pseudo-elements, we could also potentially just pull these into their own regular element (this doesn't need to be a pseudo-element)... at least in the future.. I think using `inline-inset-start` works fine for now in my opinion.